### PR TITLE
Update layout

### PR DESCRIPTION
**Pull request by drewdrewthis**
- Removes automatic padding from `Layout` component, and makes padding a `bool` optional prop.
- Removes `fullwidthAll` and just passes `fullwidth` down to `Cols`
- Gives `padding` option to `Cols`
- Augments tests to handle ^^

### CODE REVIEW

File: src/atoms/Layout/Readme.md [hunk 1]
```diff
@@ -2,29 +2,32 @@ Layout is broken out into a 12 column grid of `Col`s.
...
-  <Layout smallCols={[ 6 ]}>
+  <Layout>
     <Button variant='info'>I am button</Button>
-    <Button variant='outline'>some button</Button>
+    <Col smallCols={3}>
+      <Button variant='outline'>some button</Button>
+    </Col>
   </Layout>
 
   // Good
```
Comment (patrickkim) [inline src/atoms/Layout/Readme.md]: 👍 makes more sense

File: src/atoms/Layout/index.js [hunk 2]
```diff
@@ -151,37 +64,43 @@ Layout.propTypes = {
...
-  fullwidth: PropTypes.bool,
+  padding: PropTypes.bool,
+
   /**
-   * prevents Layout from passing left and right padding to all of its direct children (`<Cols />`).
+   * prevents Layout from automatically passing left and right padding to all of its direct children (`<Cols />`).
    * Will not remove padding that is explicitly added to a `<Col />`
    */
-  fullwidthAll: PropTypes.bool,
+  fullwidth: PropTypes.bool,
```
Comment (patrickkim) [inline src/atoms/Layout/index.js]: you missed a spot.

Comment (drewdrewthis) [inline src/atoms/Layout/index.js]: @patrickkim Explain?

Comment (patrickkim) [inline src/atoms/Layout/index.js]: nvm

**Pull request merged**